src/praisonai/praisonai/cli/commands/unified.py

```
@@ -0,0 +1,575 @@
+                    detail=f"{service} did not become ready within 15 seconds. Check {log_file}"
+                )
+            
+            # Track process for cleanup only after successful startup
+            _ACTIVE_PROCESSES.add(proc)
+            _PROCESS_LOG_HANDLES[proc] = log_handle
+            
+            return {"success": True, "message": f"Started {service} on port {service_port}"}
+            
+        except HTTPException:
+            raise
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)